### PR TITLE
Updated branch and load methods + added unit tests

### PR DESCRIPTION
**Pull request by kylercressall**
Added four unit test files for branch, branchneg, branchzero and load.

Updated each of those functions to handle more edge cases and throw errors.

Main() was changed so instead of always adding to the pointer to go to the next instruction, it was added to every method except the branch methods to give them more control where they branch to.

### CODE REVIEW

File: Milestone 2/UVSim.py [hunk 1]
```diff
@@ -110,27 +126,35 @@ def main():
             case "+10":
                 print("read")
                 read(location)
+                pointer += 1
```
Comment (j-c-williams) [inline Milestone 2/UVSim.py:129]: Good Idea

**Review by j-c-williams — APPROVED** [review]
